typescript/test/data/redemption.ts

```diff
@@ -0,0 +1,484 @@
+import { BigNumber, BytesLike } from "ethers"
```

Comment:
I think we can improve the naming of datasets:
```
singleP2PKHAddressRedemption -> singleP2PKHRedemptionWithChange
singleP2WPKHAddressRedemption -> singleP2WPKHRedemptionWithChange
singleP2SHAddressRedemption -> singleP2SHRedemptionWithChange
singleP2WSHAddressRedemption -> singleP2WSHRedemptionWithChange
multipleAddressesRedemption -> multipleRedemptionsWithChange
noChangeRedemption -> multipleRedemptionsWithoutChange
p2PKHChangeRedemption -> I think it is redundant to singleP2PKHAddressRedemption so let's remove it
```
Does it makes sense?

Reply:
Done in d510584750db537a224f7da442247f318680551d.